### ISSUE TITLE
7.3 issue - sound related (?)

### ISSUE THREAD

**Issue by Hazblade** [Auctionator/Auctionator#272]
Hey buddy. Firstly thanks for the addon. Been using it forever.

Here's an error capture for the 7.3 error. They updated the sound library. Bunch of addons having similar issues. http://imgur.com/kjZxxy5
I'm gonna replace the sound with a random working one for now.

**Comment by jrob8577** [comment]
Shoot.  Can you do a PR? I won't be home until like midnight (PST) tonight to be able to release a new version.

**Comment by Hazblade** [comment]
Yeah I had to step out for a bit but I'll do it as soon as I get back to the keyboard

**Comment by jrob8577** [comment]
I appreciate it! Once a PR is approved, the build scripts will generate a release that we can test (though I don't have any testers hanging out in the slack room ATM), and I can probably release from the office. :)

**Comment by Hazblade** [comment]
Any branch in particular?

**Comment by Hazblade** [comment]
It's super dirty and still errors in the bg but functionality is 100% as far as I can tell

**Comment by jrob8577** [comment]
I'm a fan of the standard OSS model: fork and PR from fork. I like branch names that start with the ticket number just so stuff is easy to track. 😄

**Comment by Hazblade** [comment]
Let me know if I did that right. I'm pretty new at this 🤷

**Comment by jrob8577** [comment]
Lol I glanced at it and I'm honestly not sure.  😛   Did I have changes I hadn't merged into master for a previous update? Or did you update the dependencies, too?

**Comment by Hazblade** [comment]
That's about where my knowledge ends. I just updated the auctionator.lua file and pasted that

**Comment by jrob8577** [comment]
OK. I'll take a look at it late lunch and see what can be seen.  :) .

**Comment by jrob8577** [comment]
Also @Hazblade if you PM me your email address, I can invite you to the Auctionator slack channel. It's not terribly exciting, but it's where the builds get dropped, and it's a little easier to get my attention there. :)

**Comment by Hazblade** [comment]
It's just this user @gmail. I think my github account might be too new or something. I can't seem to be able to PM or create new branches and stuff like that. Had me a bit confused earlier

**Comment by ergh99** [comment]
@Hazblade Would you mind posting your Auctionator.lua into something like pastebin.com and then posting the link here?  Pasting it in the conversation of a pull request mangled the text.

**Comment by jrob8577** [comment]
Fix available - if you log into slack with the invite sent earlier, the beta release is in the #builds channel (link in channel description).  I'll try to test myself shortly.
